app/webpacker/css/darkswarm/checkout.scss

```diff
@@ -108,3 +108,13 @@ checkout {
     }
   }
 }
+
+.not-paid {
+  color: #f40f0f;
+  font-weight: 700;
+}
+
+.paid {
+  color: black;
+  font-weight: 400;
+}
```

Comment:
From what I can see, the text was already styled this way, so I think this class isn't actually needed. Will it still look right if you remove this?

Reply:
Removed styling.